test-utils/eslint-plugin/use-vitest-local-expect.test.js

```diff
@@ -0,0 +1,66 @@
+import useVitestLocalExpect from './use-vitest-local-expect.cjs';
+
+import { createRuleTester } from 'eslint-vitest-rule-tester';
+import { describe, it } from 'vitest';
+
+describe('use-vitest-local-expect', () => {
+  const { valid, invalid } = createRuleTester({
+    name: 'use-vitest-local-expect',
+    rule: useVitestLocalExpect,
+    configs: {
+      // flat config options
+      languageOptions: {
+        parserOptions: {
+          ecmaVersion: 2020,
+          sourceType: 'module',
+        },
+      },
+    },
+  });
+
+  it('valid case', () => {
+    valid(
+      "it('should do something', ({expect}) => { expect(true).toBe(true); });",
+    );
+  });
+
+  it('invalid case: no expect accessed from context', async ({ expect }) => {
+    const { result } = await invalid({
+      code: "it('should do something', () => { expect(true).toBe(true); });",
+      errors: ['correctExpect'],
+    });
+
+    expect(result.output).toMatchSnapshot();
```

Comment:
Would inline snapshots be more useful here? Makes it easy to compare the input and expected values at a glance, especially since the output is so small.